src/docs/publish_docs.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash -ex
+
+trace_suspend() {
+    case "$-" in *x*)
+        set +x
+        __resume_trace=1
+        ;;
+    esac 2>/dev/null
+}
+
+trace_resume() {
+    if [ "${__resume_trace:-0}" = 1 ]; then
+        unset __resume_trace
+        set -x
+    fi
+}
+
+DOCS_GIT_URL='https://git.launchpad.net/~ubuntu-docker-images/ubuntu-docker-images/+git/templates'
+image_name="${1}"
+image_doc_filename="${2}"
+image_doc_folder="${3}"
+err=""
+
+AVAILABLE_ARCHS="$(yq -r '.releases[].architectures[]' < "${image_doc_folder}"/"${image_doc_filename}".yaml | sort | uniq | paste -sd ',')"
```

Comment:
is isn't `.yaml` part of the image_doc_filename?

Reply:
I prefer like this to avoid removing the .yaml when we send the mardown file to Dockerhub or ECR

Reply:
but this still poses the same pitfall as the previous comments, i.e. if the upstream job changes, this breaks.

Reply:
https://github.com/canonical/oci-factory/actions/runs/5292499620/jobs/9579455161